taxi/forms.py

```diff
@@ -0,0 +1,63 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import (
+    UserCreationForm,
+    UserChangeForm
+)
+from taxi.models import (
+    Driver,
+    Car
+)
+
+CHAR_AMOUNT = 8
+
+
+class Validator:
+    def __init__(self):
+        self.cleaned_data = None
```

Comment:
The `Validator` class sets `self.cleaned_data` to `None`, but this attribute is typically set by Django's form processing. You should not initialize it in the constructor. Instead, ensure that `clean_license_number` is used within a form where `cleaned_data` is properly populated by Django.